.github/workflows/stale-issues.yml

```
@@ -43,17 +43,18 @@ jobs:
             Please reopen if you'd like to work on this further.
           days-before-pr-stale: 14
           days-before-pr-close: 14
-          stale-pr-message: "This PR is stale because it has been open for 28 days with no activity."
+          stale-pr-message: "This PR is stale because it has been open for 14 days with no activity.It will be closed if no further activity occurs. Thank you."
+          close-pr-message: "This PR was closed because it has been inactive for 14 days since being marked as stale.Please reopen if you'd like to work on this further."
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Contribution issues
         uses: actions/stale@v5
         with:
           days-before-issue-stale: 180
-          days-before-issue-close: 365
+          days-before-issue-close: 7
```

Why did you change this?
```suggestion
          days-before-issue-close: 365
```